health/micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -0,0 +1,16 @@
+# Changelog
+
+## 0.3.0 - 2024-12-03
+#### Base Admin console & microplan web
+  1. Campaign Manager Module CSS
+  2. Boundary Manager Module CSS
+  3. Microplan Module CSS
+  4. Other Core Override CSS
+
+
```

Markdownlint (0.35.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)